### PR TITLE
get gene list from args AND from submitted form

### PR DESCRIPTION
**Pull request by northwestwitch**
This should fix https://github.com/Clinical-Genomics/scout/issues/1155, I think.

### CODE REVIEW

**Comment by northwestwitch** [comment]
I don't know if removing python 2.x and 3.x up to 3.6 is the way to go, but it kept failing the build. This way   is working..!

**Comment by ingkebil** [comment]
Removal of python 2 support is fine, we're trying to avoid running anything in production with python 2.

**Comment by ingkebil** [comment]
Do you see a way around the complicated travis setup? as in: not installing miniconda from scratch to get this to work.

**Comment by northwestwitch** [comment]
I wonder what might happen if you test it without the miniconda thing. On my local computer it just passes the test..

**Comment by ingkebil** [comment]
You can have a look at genotype, we just fixed something similar there: https://github.com/Clinical-Genomics/genotype/pull/26/files

No need to fix this in this PR btw! Just in case you feel you need to dot the i's

**Comment by northwestwitch** [comment]
Ok, thanks!

**Comment by northwestwitch** [comment]
@ingkebil or @patrikgrenfeldt could you review this PR when you have time? 
I'm not concerned about the change I made in the views, which should fix the actual bug in scout. It's more about how I removed the support for python older than 3.6. Is it professional enough? :)
The idea is that we are going to re-write this repo in a near future so I wouldn't spend so much time on making it cool..
Thanks!

File: .travis.yml [hunk 1]
```diff
@@ -3,12 +3,11 @@ sudo: false
...
 python:
-  - "2.7"
-  - "3.5"
+  - "3.6"
 
 install:
   # update periodically - saves downloading if version is the same
-  - if [[ "${TRAVIS_PYTHON_VERSION}" == "2.7" ]]; then
+  - if [[ "${TRAVIS_PYTHON_VERSION}" == "3.6" ]]; then
       wget http://repo.continuum.io/miniconda/Miniconda-3.8.3-Linux-x86_64.sh -O miniconda.sh;
```
Comment (ingkebil) [inline .travis.yml]: You're downloading miniconda for python 2 here.

You can remove the if statement and only download Miniconda3

**Comment by ingkebil** [comment]
Looking good! :+1: 

I'm still a bit puzzled as to why conda in the travis build is involved at all and why this wouldn't be solvable with a simpler travis installation.

Open to suggestions :)

**Comment by northwestwitch** [comment]
Thank you for the review! Regarding the convoluted way to test the code, I'll leave it for know, because I think @adrosenbaum will be working on this soon and the whole repo will become obsolete. I just need to fix it so keeps doing what it has been doing so far.

**Pull request merged**